faidx.c

```diff
@@ -163,7 +251,7 @@ static int write_output(faidx_t *faid, output *out, const char *name, const int
                         const char *pos_strand_name, const char *neg_strand_name,
                         enum fai_format_options format) {
     hts_pos_t seq_len, wrap_len = length, len = 0;
-    char *seq =  NULL, *qual = NULL;
+    char *seq =  NULL, *qual = NULL, *tmp = NULL;
```

Comment:
There's no need to initialise tmp to NULL here, and infact it could simply be moved down to the one block where it's used.